.env

```diff
@@ -10,7 +10,7 @@
 
 ############### Default settings ###############
 BACKEND_PORT=8080
-IMAGE_REPOSITORY=quay.io/opendatahub/odh-dashboard:nightly
+IMAGE_REPOSITORY=quay.io/maistra-dev/odh-dashboard:maistra-dev
```

Comment:
I think this we could simply set while running the builds on our end (`IMAGE_REPOSITORY=quay.io/maistra-dev/odh-dashboard:maistra-dev npm run ...`). One thing less to worry about when raising PR upstream.

Reply:
One option could be to have `maistra.env` file in the repository and use `DEFAULT_ENV_FILE` for our builds.

Reply:
:+1:  reverted this change